### PR TITLE
Tool installion check

### PR DESCRIPTION
**Pull request by dikshit1805**
In bug 0000686 and 0000687 the errors reported were due to packages not being installed.
Test scripts have been modified to first check if the tool is installed and provide installation instructions
if the tool is found not to be installed.

### CODE REVIEW

File: testing/test_scripts/check_install.sh [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# This script tests the if a perticular tool is installed or not
+
+#Usage:
+# $ sh check_install.sh <tool_name>
+
+################################################################################
+
+#!/bin/bash
+
+if [ `which $1 2>&1 >/dev/null | grep -c no` -eq 1 ]
```
Comment (sjtuhjh) [inline testing/test_scripts/check_install.sh]: How to make sure that it MUST be equal to 1 if it does not exist? It is suggested to use some simpler solution, such as [ -z "$(which $1 2>/dev/null)" ].

Comment (dikshit1805) [inline testing/test_scripts/check_install.sh]: yes, thats a simpler solution. I have sent another pull request with this change . Please check.
